source/reference/template-apps.txt

```diff
@@ -114,60 +114,83 @@ Template Apps Available
 
 .. list-table::
    :header-rows: 1
-   :widths: 30 30 50
+   :widths: 25 25 50
    
-   * - Name
-     - ID
+   * - ID
+     - Name
      - Description
 
-   * - Manage Database Views
-     - ``triggers``
+   * - ``triggers``
+     - Manage Database Views
      - Event-driven :ref:`Database Trigger <database-trigger>` template to update a view in a separate collection. Does *not* include a front-end app.
 
-   * - Flutter + Realm Flutter SDK + Atlas Device Sync Starter (Flexible Sync)
-     - ``flutter.todo.flex``
+   * - ``flutter.todo.flex``
+     - Realm Flutter SDK + Atlas Device Sync Starter (Flexible Sync)
      - Cross-platform to-do list app for Android, iOS, Windows, MacOS, and Linux written in Dart using the :ref:`Realm Flutter SDK <flutter-intro>`. Syncs local data to MongoDB Atlas using Device Sync with :ref:`Flexible Sync <flexible-sync>`. 
 
-   * - React.JS + Realm Web SDK Boilerplate
-     - ``web.mql.todo``
+   * - ``web.mql.todo``
+     - Realm Web SDK + React Starter
      - Hosted to-do list web app using the :ref:`Realm Web SDK <web-intro>`.
    
-   * - GraphQL + React App Boilerplate
-     - ``web.graphql.todo``
-     - Hosted to-do list app using the :ref:`GraphQL API <graphql-api>`.
+   * - ``web.graphql.todo``
+     - Realm Web SDK + Atlas GraphQL API + React Starter
+     - Hosted to-do list app using the :ref:`Realm Web SDK <web-intro>` and the
+       :ref:`Atlas GraphQL API <graphql-api>`.
 
-   * - Kotlin SDK + Atlas Device Sync Starter (Flexible Sync)
-     - ``android.kotlin.todo.flex``
+   * - ``kotlin.todo.flex``
+     - Kotlin SDK + Atlas Device Sync Starter (Flexible Sync)
      - Android to-do list app written in Kotlin using the :ref:`Realm Kotlin SDK <kotlin-intro>`. Syncs local data to MongoDB Atlas using Device Sync with :ref:`Flexible Sync <flexible-sync>`.
 
-   * - Java SDK + Atlas Device Sync Starter (Partition-Based Sync)
-     - ``android.kotlin.todo``
+   * - ``android.kotlin.todo``
+     - Java SDK + Atlas Device Sync Starter (Partition-Based Sync)
```

Comment:
This is just a general question, but given that we are trying to position Flexible Sync first in docs, I wonder if we want to put all the FS templates at the top, and move all the PBS templates down to the bottom?

Alternately, I wonder if these should be in alpha order by ID?

It just seems like kind of a random ordering now, which was fine when we only had a couple, but stands out to me now that we're getting so many in the list.

Reply:
yeah i thought about the same ordering suggestions that you have here. 

let's alphabetize. no potential arbitrariness there. and also now that that this in reference docs section, makes more sense